Craigslist/Craigslist.Business/ListingsManager.cs

```diff
@@ -17,17 +21,20 @@ public void PublishListing(Listing listing)
 			}
 		}
 
-		public List<Listing> GetListingsByCategoryId(long? categoryId)
+		public IPagedList<Craigslist.Domain.Entities.Listing> GetListingsByCategoryId(int? page, long? categoryId)
 		{
 			using (var domain = new CraigslistDomain())
 			{
+                int pageSize = Convert.ToInt32(ConfigurationManager.AppSettings["pageSize"]);
```

Comment:
in's totally OK to use methods from class Convert, but for parsing most of standard types there're shortcuts like int.Parse(...) or long.Parse(...) or DateTime.Parse(...).. It's just shorter :)